modules/veteran/app/models/veteran/service/representative.rb

```diff
@@ -111,6 +112,28 @@ def diff(rep_data)
         end
       end
 
+      def self.get_suffix(first_name, last_name)
+        suffixes = %w[H.M H.I.H H.R.H Dr Esq Jr Sr III II I AB BA BS BE BFA BTech LLB BSc MA MS MFA LLM
+                      MLA MBA MSc MEng JD MD DO PharmD DMin PhD EdD DPhil DBA LLD EngD]
+        suffixes.detect { |suff| last_name.include?(suff) || first_name.include?(suff) }
+      end
+
+      def self.get_representatives(first_name, last_name)
+        suffix = get_suffix(first_name, last_name)
+
+        representatives = where('lower(first_name) = ? AND lower(last_name) = ?', first_name&.downcase,
+                                last_name&.downcase)
+        if representatives.blank? && suffix.present?
+          # check without suffix
+          last_name = last_name.delete(suffix).strip
+          first_name = first_name.delete(suffix).strip
+
+          representatives = where('lower(first_name) = ? AND lower(last_name) = ?', first_name&.downcase,
+                                  last_name&.downcase)
+        end
+        representatives
```

Comment:
nit: adding some whitespace between the assignment/conditional/return statement could improve readability 🤷